### PR TITLE
dev-util/pycharm-community and dev-util/pycharm-professional: version bump to 2017.3.3

### PR DESCRIPTION
**Pull request by cklaucke**
Bug: https://bugs.gentoo.org/642948

### CODE REVIEW

File: dev-util/pycharm-community/pycharm-community-2017.3.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2017 Gentoo Foundation
```
Comment (titanofold) [inline dev-util/pycharm-community/pycharm-community-2017.3.2.ebuild]: We're in 2018 now.

File: dev-util/pycharm-community/pycharm-community-2017.3.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+HOMEPAGE="http://www.jetbrains.com/pycharm/"
+SRC_URI="http://download.jetbrains.com/python/${P}.tar.gz"
+
+LICENSE="Apache-2.0 BSD CDDL MIT-with-advertising"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=">=virtual/jre-1.8"
+DEPEND=""
```
Comment (titanofold) [inline dev-util/pycharm-community/pycharm-community-2017.3.2.ebuild]: Delete empty DEPEND.

File: dev-util/pycharm-community/pycharm-community-2017.3.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+inherit gnome2-utils readme.gentoo-r1 xdg
+
+DESCRIPTION="Intelligent Python IDE with unique code assistance and analysis"
+HOMEPAGE="http://www.jetbrains.com/pycharm/"
+SRC_URI="http://download.jetbrains.com/python/${P}.tar.gz"
+
+LICENSE="Apache-2.0 BSD CDDL MIT-with-advertising"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
```
Comment (titanofold) [inline dev-util/pycharm-community/pycharm-community-2017.3.2.ebuild]: Delete empty IUSE.

File: dev-util/pycharm-professional/pycharm-professional-2017.3.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2017 Gentoo Foundation
```
Comment (titanofold) [inline dev-util/pycharm-professional/pycharm-professional-2017.3.2.ebuild]: We're in 2018 now.

File: dev-util/pycharm-professional/pycharm-professional-2017.3.2.ebuild [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+inherit gnome2-utils readme.gentoo-r1 xdg
+
+DESCRIPTION="Intelligent Python IDE with unique code assistance and analysis"
+HOMEPAGE="http://www.jetbrains.com/pycharm/"
+SRC_URI="http://download.jetbrains.com/python/${P}.tar.gz"
+
+LICENSE="PyCharm_Academic PyCharm_Classroom PyCharm PyCharm_OpenSource PyCharm_Preview"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
```
Comment (titanofold) [inline dev-util/pycharm-professional/pycharm-professional-2017.3.2.ebuild]: Delete empty IUSE.

File: dev-util/pycharm-professional/pycharm-professional-2017.3.2.ebuild [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+RDEPEND=">=virtual/jre-1.8
+	 dev-python/pip"
+DEPEND=""
```
Comment (titanofold) [inline dev-util/pycharm-professional/pycharm-professional-2017.3.2.ebuild]: Delete empty DEPEND.

File: dev-util/pycharm-community/pycharm-community-2017.3.2.ebuild (hunk 1 above)
Comment (cklaucke) [reply dev-util/pycharm-community/pycharm-community-2017.3.2.ebuild]: fixed in  07fa91a

File: dev-util/pycharm-community/pycharm-community-2017.3.2.ebuild (hunk 3 above)
Comment (cklaucke) [reply dev-util/pycharm-community/pycharm-community-2017.3.2.ebuild]: fixed in  07fa91a

File: dev-util/pycharm-community/pycharm-community-2017.3.2.ebuild (hunk 2 above)
Comment (cklaucke) [reply dev-util/pycharm-community/pycharm-community-2017.3.2.ebuild]: fixed in  07fa91a

File: dev-util/pycharm-professional/pycharm-professional-2017.3.2.ebuild (hunk 4 above)
Comment (cklaucke) [reply dev-util/pycharm-professional/pycharm-professional-2017.3.2.ebuild]: fixed in 1ca1cca

File: dev-util/pycharm-professional/pycharm-professional-2017.3.2.ebuild (hunk 5 above)
Comment (cklaucke) [reply dev-util/pycharm-professional/pycharm-professional-2017.3.2.ebuild]: fixed in 1ca1cca

File: dev-util/pycharm-professional/pycharm-professional-2017.3.2.ebuild (hunk 6 above)
Comment (cklaucke) [reply dev-util/pycharm-professional/pycharm-professional-2017.3.2.ebuild]: fixed in 1ca1cca

**Comment by titanofold** [comment]
We probably want these squashed to two commits (one for each ebuild).

**Comment by cklaucke** [comment]
Squashed!

**Comment by kapsh** [comment]
https://blog.jetbrains.com/pycharm/2018/01/pycharm-2017-3-3-out-now/
2017.3.3 arrived already. Could you update your PR?

**Comment by cklaucke** [comment]
Sure.

**Pull request closed without merging**